org.eclipse.transformer/src/main/java/org/eclipse/transformer/Transformer.java

```diff
@@ -1010,17 +1023,20 @@ public boolean setOutput() {
 
 		} else {
 			int indexOfLastSlash = inputName.lastIndexOf('/');
```

Comment:
It appears we don't need `indexOfLastSlash` any more, so this line can be removed.